.lintstagedrc

```diff
@@ -1,7 +1,12 @@
 {
   "$schema": "http://json.schemastore.org/lintstagedrc.schema",
   "**/*.{js, ts}": [
+    "prettier --write",
```

Comment:
？？？

Reply:
`.eslintrc.js` are ignored by eslint by default.

Reply:
Then add `!.eslintrc.js` in `.eslintignore` would be better to me.

And I'm always using `autofix: true` in VSC.